content/klc/S6.2.adoc

```diff
@@ -1,9 +1,27 @@
 +++
-brief = "Component fields must be filled appropriately"
+brief = "Symbol and alias fields and metadata filled out as required"
 +++
 
 . *Reference* field is selected appropriately for the symbol and is _visible_
 . *Value* field contains the name of the symbol and is _visible_
-. *Footprint* field contains footprint link for atomic parts, and is empty for generic parts. It must be _invisible_
-. *Datasheet* field is left empty, and is _invisible_
-. The symbol contains no other custom fields
\ No newline at end of file
+. *Footprint* field contains footprint link for fully specified symbols and must be _invisible_
+.. The footprint field must be empty for generic symbols
+. *Datasheet* entry [1] is filled out, and is _invisible_
+. The symbol contains no other custom fields
+
+Additional documentation is provided via two other fields:
+[start=6]
+. **Description** field contains comma-separated device information
+.. For symbols with a default footprint, the simplified footprint name should be appended to the description e.g. `SOIC-8`
+.. Part name should not be duplicated in the description field
+. **Keywords** field contains space-separated keyword values. These values are used to assist in part searching and should not include filler words
+
+{{< klcimg src="symbol_metadata" title="Symbol metadata example" >}}
+
+**Aliases** define their own values for the datasheet link, description and keyword entries.
+
+{{< klcimg src="alias_metadata" title="Alias metadata example" >}}
+
+---
+
+_[1]: KiCad 5.x and earlier have two places to store the datasheet link. It can be in the `.lib` file once per symbol or in the `.dcm` file allowing different datasheets per alias. Having different datasheets for aliases only works if the `.lib` entry is left empty. The datasheet entry in the main symbol metadata view is linkded to the dcm entry since version 5.1.x._
```

Comment:
Typo of `linked` in the last sentence. Remove the first `d` in `linkded`.

Reply:
fixed